pages/components/EventCard/index.js

```diff
@@ -7,13 +7,13 @@ const StyledArticle = styled.article`
   width: fit-content;
 `;
 
-export default function EventCard({ title, date, type, desc }) {
+export default function EventCard({ title, date, type, description }) {
   return (
     <StyledArticle>
       <h3>{title}</h3>
       <p>{date}</p>
       <p>{type}</p>
-      <p>{desc}</p>
+      <p>{description}</p>
```

Comment:
From a semantic point of view: Since `date` and `type` look more like some kind of meta-info, a `<span>` or list could be a better solution instead of a paragraph (which totally  fits for the `description`).

Reply:
paragraph looks better than <span> or list so left it as it is